dss/operations/util.py

```diff
@@ -0,0 +1,60 @@
+
+import os
+import traceback
+import typing
+from uuid import uuid4
+from contextlib import AbstractContextManager
+
+from cloud_blobstore import BlobStore
+
+from dss.util.aws.clients import sqs, sts
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+
+_account_id = sts.get_caller_identity()['Account']  # type: ignore
+_queue_url = "https://sqs.{}.amazonaws.com/{}/dss-operations-{}".format(os.environ['AWS_DEFAULT_REGION'],
+                                                                        _account_id,
+                                                                        os.environ['DSS_DEPLOYMENT_STAGE'])
+
+
+class CommandForwarder(AbstractContextManager):
+    """
+    Context manager for forwarding commands for Lambda execution through batch SQS queuing
+    """
+    def __init__(self):
+        self.chunk = list()
+
+    def forward(self, msg: str):
+        self.chunk.append(msg)
+        if 10 == len(self.chunk):
```

Comment:
Why groups of ten?  Also... is there any danger of commands getting hung up if this takes too long to reach `__exit__`?

Reply:
The maximum number of messages that can be batch delivered to SQS is 10.

I suppose if SQS is unavailable this could get hung up, but that's unrelated to the context manager logic.